plutus-tx-plugin/test/Plugin/Laziness/lazyDepUnit.plc.golden

```diff
@@ -1,16 +1 @@
-(program
```

Comment:
Here's an example of where optimizing these tests are bad: this test is kind of pointless if we optimize it away!

Reply:
Well, we could disable the optimisations for this particular test case...

Reply:
Disabled simplifier on `plutus-tx-test` and added at least one pass of DCE to make the code readable.